assets/i18n/en/database_maps.json

```diff
@@ -61,8 +61,7 @@
   "copy_tiled_files": "Copy the Tiled maps to the project folder",
   "add_missing_rmxp_maps": "Addition of missing RPG Maker XP maps",
   "create_new_maps": "Creating new maps",
-  "import_success_message_singular": "The map has been successfully imported.",
-  "import_success_message_plurial": "The maps has been successfully imported.",
+  "import_success_message": "You can find your maps and associated resources in the Data/Tiled folder for your project.",
```

Comment:
in the Data/Tiled folder of* your project

Reply:
Corrigé. :)